src/cpp/rtps/participant/RTPSParticipantImpl.cpp

```diff
@@ -62,6 +62,7 @@
 #include <rtps/persistence/PersistenceService.h>
 #include <statistics/rtps/GuidUtils.hpp>
 
+
```

Comment:
Please, revert this change as it is not including nothing new to the PR.

Reply:
removed in [ee520e4](https://github.com/eProsima/Fast-DDS/pull/4177/commits/ee520e48f15c205983ef93344d678cca12d35d0b)